core/src/com/unciv/logic/automation/ChooseBeliefsAutomation.kt

```diff
@@ -155,31 +176,36 @@ object ChooseBeliefsAutomation {
                 UniqueType.BuyUnitsByProductionCost ->
                     15f * if (civInfo.wantsToFocusOn(Victory.Focus.Military)) 2f else 1f
                 UniqueType.StatsWhenSpreading ->
-                    unique.params[0].toInt() / 5f
+                    unique.params[0].toFloat() / 5f
                 UniqueType.StatsWhenAdoptingReligion, UniqueType.StatsWhenAdoptingReligionSpeed ->
                     unique.stats.values.sum() / 50f
                 UniqueType.RestingPointOfCityStatesFollowingReligionChange ->
-                    unique.params[0].toInt() / 7f
+                    if (civInfo.wantsToFocusOn(Victory.Focus.CityStates))
+                        unique.params[0].toFloat() / 3.5f
+                    else
+                        unique.params[0].toFloat() / 7f
                 UniqueType.StatsFromGlobalCitiesFollowingReligion ->
-                    50f / unique.stats.values.sum()
+                    unique.stats.values.sum()
+                UniqueType.StatsFromGlobalFollowers ->
+                    4f * (unique.stats.values.sum() / unique.params[1].toFloat())
                 UniqueType.StatsSpendingGreatPeople ->
                     unique.stats.values.sum() / 2f
                 UniqueType.Strength ->
-                    unique.params[0].toInt() / 4f
+                    unique.params[0].toFloat() / 4f
                 UniqueType.ReligionSpreadDistance ->
-                    (10 + unique.params[0].toInt()) / goodEarlyModifier
+                    (10f + unique.params[0].toFloat()) * goodEarlyModifier
                 UniqueType.NaturalReligionSpreadStrength ->
-                    (10 + unique.params[0].toInt()) / goodEarlyModifier
+                    unique.params[0].toFloat() * goodEarlyModifier / 5f
                 UniqueType.SpreadReligionStrength ->
-                    unique.params[0].toInt() / goodLateModifier
+                    unique.params[0].toFloat() * goodLateModifier / 5f
                 UniqueType.FaithCostOfGreatProphetChange ->
-                    unique.params[0].toInt() / goodLateModifier / 2f    
+                    -unique.params[0].toFloat() * goodLateModifier / 2f
                 UniqueType.BuyBuildingsDiscount, UniqueType.BuyItemsDiscount, UniqueType.BuyUnitsDiscount ->
-                    unique.params[2].toInt() / goodLateModifier
+                    -unique.params[2].toFloat() * goodLateModifier / 5f
```

Comment:
Added some subjective scaling weights since these boni tend to be larger numerically compared to the stat-providing uniques (e.g., `[100]%` vs `[+1 Gold] from every global follow`)